layers/sessions.go

```diff
@@ -76,7 +76,7 @@ func (s *SessionsLayer) onResponseError(state *httransform.LayerState) {
 		close(channelUntyped.(chan<- string))
 	}
 
-	brokenSessionID, _ := state.ResponseHeaders.GetString("x-crawlera-session")
+	brokenSessionID, _ := state.ResponseHeaders.GetString("zyte-proxy-session")
```

Comment:
zyte-proxy-session requires implementation on proxy side. We have to keep **x-crawlera-session** here.